libs/pika/execution_base/tests/unit/any_sender.cpp

```diff
@@ -696,6 +626,59 @@ void test_when_all()
     tt::sync_wait(std::move(as3));
 }
 
+// To control when the move constructor will throw, exception is caught in the set_value of
+// any_receiver
+bool throwing = false;
+
+struct throwing_constructor
+{
+    throwing_constructor() noexcept(false) {}
+    throwing_constructor(throwing_constructor&&) noexcept(false)
+    {
+        if (throwing) throw std::runtime_error("error");
+    };
+    throwing_constructor(const throwing_constructor&) = default;
+};
+
+void test_throwing_constructor()
+{
+    {
+        // With unique_any_sender
+        throwing = false;
+        std::atomic<bool> set_error_called{false};
+        ex::unique_any_sender<throwing_constructor> send_unique_any_ =
```

Comment:
Could you please update the variable names here (and further down) to be consistent with the other variables in this file? Especially no trailing `_` please.